cypress/integration/gift_card_add_spec.ts

```diff
@@ -0,0 +1,29 @@
+/// <reference types="Cypress" />
+
+import {GiftCardsPo} from "../support/giftCards.po";
+import {HomePo} from "../support/home.po";
+import {GiftCardPo} from "../support/giftCard.po";
+
+describe ('Choosing gift card and adding it to card', () => {
+    const homePage = new HomePo();
+    const giftCardsPage = new GiftCardsPo();
+    const giftCardPage = new GiftCardPo();
+
+
+    beforeEach( () => cy.visit(homePage.pageUrl));
+
+        it('Selecting Gift Card and adding it to the card', () => {
+           const price = '50';
+           const addedToCardText = 'Added to Cart';
+
+           homePage.clickOnElem(homePage.giftCardBtn);
+           cy.url().should('include', giftCardsPage.pageUrl);
+           giftCardsPage.isElemVisible(giftCardsPage.imgGiftLink);
```

Comment:
unnecessary check, because if this elem will be not visible, you could not click on it